src/Avalonia.Visuals/Media/Color.cs

```diff
@@ -306,5 +309,61 @@ public override int GetHashCode()
         {
             return !left.Equals(right);
         }
+#if !BUILDTASK
+        /// <summary>
+        /// Get the System Accent Color, if the Operating System is not supported, returns #FF0078D7
+        /// </summary>
+        /// <returns>The accent color</returns>
+        public static Color GetSystemAccentColor()
+        {
+            var isWindows10_8 = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) & 
+                            !(Environment.OSVersion == new OperatingSystem(PlatformID.Win32NT, new Version(10,0)) ||
+                             Environment.OSVersion == new OperatingSystem(PlatformID.Win32NT, new Version(6,3))); //check if the OS is Windows 10 or Windows 8
+            var defaultValue = Color.DefaultAccentColor;// default Avalonia Accent Color
+
+            //TODO: Add Support for MacOS X Accent color and Linux Desktop Envs with this feature
+            if(isWindows10_8)
+            {
+                return Avalonia.Win32.AccentColor.GetAccentColorWin("ImmersiveSystemAccent");
+            }
+            else
+            {
+                return Color.Parse(defaultValue);
+            }
+        }
+#endif
+
+        /// <summary>
+        /// Change the luminosity of a color.
+        /// </summary>
+        /// <param name="color">The color to change its luminosity</param>
+        /// <param name="newluminosityFactor">The new Luminosity for the Color</param>
+        /// <returns>A new Color with the Luminosity Factor applied.</returns>
+        public static Color ChangeColorLuminosity(Color color, double newluminosityFactor)
```

Comment:
Is this some well-defined known algorithm? If so, please, add a name/link. Otherwise it shouldn't be a public method of the Color type

Reply:
@PieroCastillo I would also keep this method internal inside of AccentColors class. We will be always able to make it public if it's needed.

Reply:
well, I'm going to do it.